### DISCUSSION TITLE
Disable the bootloader backend

### DISCUSSION THREAD

**Discussion by Louson** [rauc/rauc#994] [Q&A] [answered]
rauc 1.8
building with yocto kirkstone

Suppose you do not want any bootloader backend (at least for some tests). If I don't set the bootloader in `system.conf`, it fails with

`rauc[391]: Failed to initialize context: Failed to load system config (/etc/rauc/system.conf): No custom bootloader backend defined`

I tried to set `custom`, but it asks for a custom backend:
`rauc[391]: Failed to initialize context: Failed to load system config (/etc/rauc/system.conf): No custom bootloader backend defined`

I would like to be not define a bootloader, or set it to 'none'.

**Comment by ejoerns** [comment, marked as answer]
The `custom` bootloader requires a `bootloader-custom-backend=` in the system.conf's `[handlers]` section:

``` ini
[handlers]
...
bootloader-custom-backend=custom-bootloader-script
```

Maybe you are searching for the `noop` bootloader?

What is your use case for not even wanting a custom backend?

**Comment by Louson** [comment]
Indeed! `noop` makes it. I'm sorry, I missed it in the documentation and it does not appear in `config_file.c`

Sorry for that !

**Reply by ejoerns** [reply]
No need to excuse, this is what discussions are for :wink:
